app/controllers/documents_controller.rb

```diff
@@ -1,5 +1,20 @@
 class DocumentsController < ApplicationController
-  before_action :require_intake
+  include AccessControllable
+  include FileResponseControllerHelper
+
+  before_action :require_sign_in, :require_beta_tester, only: [:index, :show]
+  before_action :require_intake, only: [:destroy]
+  layout "admin", only: [:index]
+
+  def index
+    @client = Client.find(params[:client_id])
+    @documents = @client.documents
+  end
+
+  def show
+    @document = Document.find(params[:id])
+    render_active_storage_attachment @document.upload
+  end
 
   def destroy
     document = current_intake.documents.find_by(id: params[:id])
```

Comment:
❓ I was initially confused to see that the intake-related `destroy` lives in the same controller as `client`-related `index` and `show`!

👍 It's also OK. I foresee confusion when we add a client document view delete button. We can resolve any such confusion down the road.

Reply:
a good point! we should probably namespace everything going into case management soon